test/execute_build_test.go

```diff
@@ -469,6 +472,618 @@ func testCreateImage(t *testing.T, when spec.G, it spec.S) {
 	})
 }
 
+func testSignBuilder(t *testing.T, _ spec.G, it spec.S) {
```

Comment:
I've moved it to `cosign_e2e_test.go` in the same package, let me know your thoughts!

Reply:
I like it, it's annoying that we have to redefine all the `it.Before` and `it.After` again, but maybe we can look into a way of refactoring that in a future PR

Reply:
On that note, perhaps when this refactoring happens tests could be split using Go build constraint tags?
I've seen that pattern before where e2e tests are tagged as such using `//go:build e2e` (or something to that effect). That would allow unit tests to be run using `go test -v ./...`, as well as running e2e tests with something like `go test -v -tags=e2e ./...`.